### PR TITLE
Add delete logic

### PR DESCRIPTION
**Pull request by saifudinbadani**
Logic added for adding and deleting notes.

### CODE REVIEW

File: src/components/HomePage/MainContent.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    // }
+    // const saveNotesHandler = () => {
+    //     setNotesData((prev) => [...prev, notes])
+        
+    //     setNotes({
+    //         title: '',
+    //         content: '',
+    //     })
+       
+    // }
```
Comment (vaishnavipaliya11) [inline src/components/HomePage/MainContent.js:27]: remove commented code

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    return  <nav className="nav-container display-flex primary-clr p-rl-6">
+    <h1 to='/' className="nav-link font-size-2pt6 heading-1">Jotter</h1>
+    <div className="input-search pos-rltv m-rght-auto">
+        <i className="fas fa-filter pos-absolute font-size-2pt6"></i>
+        <input type="text" placeholder="search" className="nav-input-search"/>
+    </div>
+    {/* <Link to='/productlist' className="nav-link-cta">Shop Now</Link>
+    <Link to='/login' className="nav-link"><button
+            className="btn btn-solid-secondary">Login</button></Link> */}
```
Comment (vaishnavipaliya11) [inline src/components/Navigation/Navigation.js:14]: remove commented code

**Review by vaishnavipaliya11 — APPROVED** [review]
Pease don't make so many changes in single PR. Remove commented code. rest looks good.

**Review by yogesh- — APPROVED** [review]
- code looks clean
- small components
can be merged

**Pull request merged**